app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -97,19 +101,36 @@
     </div>
 
     <div class="form-group" >
-      <label class="required">Access Mode</label><br/>
-      <label class="radio-inline">
-        <input type="radio" name="accessModes" ng-model="claim.accessModes" value="ReadWriteOnce" aria-describedby="access-modes-help" ng-checked="true" >
-        Single User (RWO)
-      </label>
-      <label class="radio-inline">
-        <input type="radio" id="accessModes" name="accessModes" ng-model="claim.accessModes" value="ReadWriteMany" aria-describedby="access-modes-help" >
-        Shared Access (RWX)
-      </label>
-      <label class="radio-inline">
-        <input type="radio" name="accessModes" ng-model="claim.accessModes" value="ReadOnlyMany" aria-describedby="access-modes-help">
-        Read Only (ROX)
+      <label class="required">Access Mode
+        <span ng-if="claim.storageClass && (claim.storageClass | storageClassAccessMode)">
+          <small>(cannot be changed)</small>
+        </span>
       </label>
+      <div>
+        <label class="radio-inline">
+          <input type="radio" name="accessModes" ng-model="claim.accessModes" value="ReadWriteOnce"
+                 aria-describedby="access-modes-help" ng-true-value="’1’" ng-false-value="’0’"
+                 ng-checked="((claim.storageClass | storageClassAccessMode)==='ReadWriteOnce')
+                  || !claim.storageClass || (claim.storageClass.metadata.name === 'No Storage Class')
+                  || !(claim.storageClass | storageClassAccessMode)"
```

Comment:
Can't this simply be

```
ng-checked="!(claim.storageClass | storageClassAccessMode) || (claim.storageClass | storageClassAccessMode)==='ReadWriteOnce')"
```

Reply:
It can now, since the "No Service Class" option is covered.  Let me test and change it.

Reply:
Ok, I had to leave in the '|| !claim.storageClass' for when nothing is chosen in the drop down.  Checking in now.